mediaflow_proxy/extractors/vixcloud.py

```
@@ -39,31 +39,32 @@ async def version(self, site_url: str) -> str:
             raise ExtractorError("Failed to extract URL components, Invalid Request")
         soup = BeautifulSoup(response.text, "lxml", parse_only=SoupStrainer("body"))
         if soup:
             script = soup.find("body").find("script").text
             token = re.search(r"'token':\s*'(\w+)'", script).group(1)
             expires = re.search(r"'expires':\s*'(\d+)'", script).group(1)
-            vixid = iframe.split("/embed/")[1].split("?")[0]
-            base_url = iframe.split("://")[1].split("/")[0]
-            final_url = f"https://{base_url}/playlist/{vixid}.m3u8?token={token}&expires={expires}"
-            if "canPlayFHD" in query_params:
-                # canPlayFHD = "h=1"
+            canPlayFHD = re.search(r"window\.canPlayFHD\s*=\s*(\w+)", script).group(1)
```

Ruff (0.11.9)

Local variable `canPlayFHD` is assigned to but never used

Remove assignment to unused variable `canPlayFHD`

(F841)